File: core/android/app/src/main/java/com/aco/fitar/api/models/AnchorModel.kt
package com.aco.fitar.api.models

import com.aco.fitar.api.ModelResponseBase
import kotlinx.serialization.Serializable

@Serializable
class AnchorModel(var sessionid:String, var anchorid:String, var lat:Double, var lng:Double, var noteText:String) {
}

@Serializable
class AnchorResponseModel : ModelResponseBase() {
}

@Serializable
class AnchorResponse {
    public var id:String = "";
    public var username:String = "";
    public var text:String = "";
}

File: core/FitAR/FitAR.API/Models/AnchorModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FitAR.Web.API.Models
{
  public class AnchorModel
  {
    public string sessionid { get; set; }
    public string anchorid { get; set; }
    public double lat { get; set; }
    public double lng { get; set; }
    public string noteText { get; set; }
  }

  public class AnchorResponseModel : ModelBase
  {
  }

  public class AnchorResponse
  {
    public string id { get; set; }
    public string username { get; set; }
    public string text { get; set; }
  }

}
